services/api-server/src/simcore_service_api_server/api/routes/files.py

```diff
@@ -97,11 +97,25 @@ async def upload_file(
     )
 
     logger.info("Uploading %s to %s ...", file_meta, presigned_upload_link)
-    async with httpx.AsyncClient(timeout=httpx.Timeout(5.0, write=3600)) as client:
-        assert file_meta.content_type  # nosec
-
-        resp = await client.put(presigned_upload_link, data=await file.read())
-        resp.raise_for_status()
+    try:
+        #
+        # FIXME: TN was uploading files ~1GB and would raise httpx.ReadTimeout.
+        #  - Review timeout config (see api/dependencies/files.py)
+        #
+        async with httpx.AsyncClient(
+            timeout=httpx.Timeout(5.0, read=60.0, write=60.0)
```

Comment:
We increase the reading timeout, but we decrease the writing one, is that correct?

Reply:
you are right. I cannot probe that this setup will be better because i cannot add tests now
i will revert the write value and increase the read value